com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -95,7 +95,8 @@ public Matrix4x4 GetGPUProjectionMatrix()
         internal float aspectRatio;
         public float renderScale;
         public bool clearDepth;
-        public bool isSceneViewCamera;
+        [Obsolete("Please use cameraType instead.", true)] public bool isSceneViewCamera;
```

Comment:
This just occurred to me, what if we make this a property instead that returns `cameraType == SceneView`? We don't consume data from struct and avoid duplicating the code in several places.